config/pepperConfig.js.ctmpl

```diff
@@ -259,3 +259,40 @@ var DDP_ENV = {
         {{end}}
     {{end}}
 {{end}}
+
+{{if eq $study_guid "circadia"}}
+    {{if eq $environment "local"}}
+        DDP_ENV['auth0ClientId'] = "GMMFtLjB1c38Fmr3UVEm8WqBsvCIeJ9d";
+        DDP_ENV['auth0Domain'] = "circadia-dev.us.auth0.com";
+        DDP_ENV['auth0Audience'] = "circadia-dev.us.auth0.com";
+        DDP_ENV['basePepperUrl'] = "https://pepper-dev.datadonationplatform.org";
+    {{end}}
+
+    {{if eq $environment "dev"}}
+        DDP_ENV['auth0ClientId'] = "tbDahmqfZo7ircGJ6AGsBrtXXGgA29Pk";
+        DDP_ENV['auth0Domain'] = "circadia-dev.us.auth0.com";
+        DDP_ENV['auth0Audience'] = "circadia-dev.us.auth0.com";
+        DDP_ENV['basePepperUrl'] = "https://pepper-dev.datadonationplatform.org";
+    {{end}}
+
+    {{if eq $environment "test"}}
+        DDP_ENV['auth0ClientId'] = "TBD";
+        DDP_ENV['auth0Domain'] = "TBD";
+        DDP_ENV['auth0Audience'] = "TBD";
+        DDP_ENV['basePepperUrl'] = "https://pepper-test.datadonationplatform.org";
+    {{end}}
+
```

Comment:
Do we have google analytics for Circadia study?

Reply:
No, not at this point.